plugins/participants.js

```diff
@@ -0,0 +1,167 @@
+module.exports = (controller) => {
+    const renderParticipantsBlocks = async (participants) => {
```

Comment:
same as `renderParticipantBlocks` - looks like this function doesn't need the `async` keyword